scrapi/base/transformer.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from lxml import etree
+
+XML_to_JSON = 'xml_to_json'
+
+supported_transformations = [XML_to_JSON]
```

Comment:
I don't like the way this is done either. Maybe I'm trying to make it too generic too early.